giskard/llm/loaders/prompt_injections.py

```diff
@@ -0,0 +1,100 @@
+import ast
+import requests
+from typing import Optional
+import pandas as pd
+
+from giskard.datasets.base import Dataset
+
+INJECTION_DATA_URL = "https://raw.githubusercontent.com/Giskard-AI/prompt-injections/main/prompt_injections.csv"
+GISKARD_META_URL = "https://raw.githubusercontent.com/Giskard-AI/prompt-injections/main/giskard_meta_data.csv"
```

Comment:
Not sure this is the best in the long term… it could break if we change the repo. Maybe it’s safer to store them on some public S3 bucket? Or point to specific tag on the prompt-injections repo.

Reply:
Once you approve the PR, I'll tag the current git version

Reply:
TODO

Reply:
done